crates/eframe/src/native/run.rs

```diff
@@ -342,132 +348,258 @@ mod glow_integration {
         painter: egui_glow::Painter,
         integration: epi_integration::EpiIntegration,
         app: Box<dyn epi::App>,
-
         // Conceptually this will be split out eventually so that the rest of the state
         // can be persistent.
         gl_window: GlutinWindowContext,
     }
+    /// This struct will contain both persistent and temporary glutin state.
+    ///
+    /// Platform Quirks:
+    /// Microsoft Windows: requires that we create a window before opengl context.
+    /// Android: window and surface should be destroyed when we receive a suspend event. recreate on resume event.
```

Comment:
```suggestion
    /// * Microsoft Windows: requires that we create a window before opengl context.
    /// * Android: window and surface should be destroyed when we receive a suspend event. recreate on resume event.
```

(markdown treats a single newline as no newline)